typings/intl-messageformat-parser/index.d.ts

```diff
@@ -1,7 +1,7 @@
 declare module 'intl-messageformat-parser' {
   interface Element {
     type: 'messageTextElement'|'argumentElement';
-    id: string
+    id?: string
     value?: string
     format?: null | {type: string; style?: string};
   }
```

Comment:
if format and id are always defined if the other is, it may make sense to make a discriminated union out of this, but fine for now (won't help with the `.filter()`s anyway)